docs/reference/triggers/kafka.md

```diff
@@ -1,110 +1,315 @@
+
+<a id="multiple-topics"></a>
+### Multiple topics
+
+Up until now, the overview discussed partitions and workers. But a Nuclio replica can also read from multiple topics. A Nuclio replica can use its workers to handle the partitions of multiple topics instead of only those of a single topic.
+For example, if your replica has 10 workers and is configured to handle 10 topics, each with 100 partitions, the replica is essentially using 10 workers to handle 1,000 partitions.
+
+<a id="config-params"></a>
+## Configuration parameters
+<!-- See https://kafka.apache.org/documentation/#consumerconfigs + types and
+  default values in pkg/processor/trigger/kafka/types.go. -->
+
+Use the following trigger attributes for basic configurations of your Kafka trigger.
+You can configure each attribute either in the `triggers.<trigger>.attributes.<attribute>` function `spec` element (for example, `triggers.myKafkaTrigger.attributes.sessionTimeout`) or by setting the matching `nuclio.io` annotation key (for example, `nuclio.io/kafka-session-timeout`); (note that not all attributes have matching annotation keys).
+For more information on Nuclio function configuration, see the [function-configuration reference](/docs/reference/function-configuration/function-configuration-reference.md).
+
+> **Note:** For more advanced configuration parameters, see the configuration sections under [How a message travels through Nuclio to the handler](#message-course-config-params) and [Rebalancing](#rebalancing-config-params). For an example, see [Configuration example](#config-example).
+
+- <a id="topic"></a>**`topic`** - The name of the topic on which to listen.
+  <br/>
+  **Type:** `string`
+
+- <a id="brokers"></a>**`brokers`** - A list of broker IP addresses.
+  <br/>
+  **Type:** `[]string`
+
+- <a id="paritions"></a>**`paritions`** - A list of partitions for which the function receives events.
+  <br/>
+  **Type:** `[]int`
+
+- <a id="consumerGroup"></a>**`consumerGroup`** - The name of the Kafka consumer group to use.
+  <br/>
+  **Type:** `string`
+
+- <a id="initialOffset"></a>**`initialOffset`** - The location (offset) within the partition at which to begin the message processing.
+  Currently, you can begin the processing either with the earliest or latest ingested messages.
+  <br/>
+  **Type:** `string`
+  <br/>
+  **Valid Values:** `"earliers" | "latest"`
```

Comment:
`earliers` -> `earliest`

Reply:
Ugh! For some reason my spell checker missed this ... . => Fixed — commit a049c04.